webknossos/Changelog.md

```diff
@@ -15,8 +15,11 @@ For upgrade instructions, please check the respective *Breaking Changes* section
 ### Breaking Changes
 
 ### Added
+- Added a new parameter to `Layer.downsample` & `layer.upsample` which takes an existing dataset and uses the magnifications of this dataset as guidance for the downsampling. [#730](https://github.com/scalableminds/webknossos-libs/pull/730)
 
 ### Changed
+- Changed the name of `max_mag` in `Layer.downsample` to `coarsest_mag`. [#730](https://github.com/scalableminds/webknossos-libs/pull/730)
+- The `Layer.downsample` and `Layer.upsample`function now automatically downsample according to magnifications already existing in the dataset. This behaviour can be turned off by setting the new parameter `align_with_other_layers` to `False`. [#730](https://github.com/scalableminds/webknossos-libs/pull/730)
```

Comment:
in my opinion, these changes are breaking changes, since the interface is not backwards-compatible when named arguments were used before. I think, we should change this in the changelog.

Reply:
Oh 😅. Looks like I placed them wrong. Did / Do you correct this @philippotto?

Reply:
Just opened a PR for that :)